newrelic/core/config.py

```diff
@@ -1371,6 +1407,8 @@ def finalize_application_settings(server_side_config=None, settings=_settings):
 
     application_settings.attribute_filter = AttributeFilter(flatten_settings(application_settings))
 
+    simplify_distributed_tracing_sampler_granularity_settings(application_settings)
```

Comment:
Since the logic for this has now moved to `apply_server_side_settings()`, the call to this function can be removed